docs/airflow/production-maintenance.md

```diff
@@ -1,8 +1,60 @@
+    * If no, this will run even if prior days have failed
+    * If **📆 yes**, this will wait for prior days to succeed before current day can succeed
+* **All of history** - Some DAGs contain only operations that cover all of history, even when only for one day (i.e., none of their tasks do anything that is specific to their `execution_date`.) So if these DAGs fail for multiple days in a row, only the latest day needs to be re-run to refresh all the data.
+    * If **🔂 no**, must re-run this DAG for each day that failed to ensure that all data is captured
+    * If yes, can re-run only the most recent date
+* **Depends on** - The information in the table below is a high-level summary indicating general DAG execution order and explicit inter-DAG dependencies; there may be additional dependencies for individual tasks within a DAG, and there may be cases where a DAG task depends on **data** generated by another DAG task but the task-task dependency is not formally specified.
+    * If N/A, this DAG is the bottom of the dependency ladder and can run independently.
+    * If populated, this DAG has upstream dependencies that must complete before it will run
+
+
+DAGs are listed in alphabetical order, as they appear in the Airflow UI.
+
+| DAG | Safe after 24h | `depends_ on_past` | All of history | Depends on | Notes |
+| --- | --- | --- | --- | --- | --- |
+`airtable_loader` | **⛔ No (all tasks)** | No | **🔂 No** | N/A | |
+`airtable_views` | Yes | No | Yes* | `airtable_loader` | Latest-only data |
+`amplitude_benefits` | Yes | No | **🔂 No** | N/A | |
+`gtfs_downloader` | **⛔ No (`generate_provider_list` and `download_data` tasks specifically)** | No | **🔂 No** | N/A | Tasks downstream of `download_data` can safely be rerun after 24 hours |
+`gtfs_loader` | Yes | No | **🔂 No** | `gtfs_downloader`* | Technically also depends on `gtfs_schedule_history`, not usually an issue |
+`gtfs_schedule` | Yes | No | Yes* | `gtfs_views_staging` | Latest-only data (but depends on `gtfs_views_staging` for data cleaning) |
+`gtfs_schedule_history` | N/A | N/A | N/A | N/A | Once-only (defines external tables); does not generally need to be re-run |
+`gtfs_schedule_history2` | Yes | **📆 Yes** | **🔂 No** | `gtfs_loader` | |
+`gtfs_views` | Yes | No | Yes | `gtfs_views_staging` | |
+`gtfs_views_staging` | Yes | No | Yes | `gtfs_schedule_history2` | |
+`payments_loader` | Yes | No | Yes | N/A | |
+`payments_views` | Yes | No | Yes | `payments_views_staging`| |
+`payments_views_staging` | Yes | No | Yes | `payments_loader` | |
+`rt_loader` | Yes | No | **🔂 No** | `gtfs_loader` | |
+`rt_loader_files` | Yes | No | **🔂 No** | N/A | |
+`rt_timestamp_fix` | N/A | N/A | N/A | N/A | DAG is deprecated but still appears in Airflow UI |
+`rt_views` | Yes | No | Yes | `rt_loader`, `gtfs_views` | |
+`sandbox` | N/A | N/A | N/A | N/A | Testing only; does not need to be re-run |
```

Comment:
The `Notes` column is kind of hard to see.

![Screen Recording 2022-04-04 at 10 39 52 AM](https://user-images.githubusercontent.com/3112493/161601098-564897f8-cb4d-4557-847c-4064a88830f3.gif)

Reply:
Moved some things around so at least part of the column is visible, if you think it's not sufficient perhaps we can make a separate issue asking Charlie about making the left sidebar narrower so that the content can take more space